fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/InvalidPrefixExceptionMapper.java

```diff
@@ -44,7 +43,6 @@
 
     @Override
     public Response toResponse(final InvalidPrefixException e) {
-        LOGGER.error("FedoraInvalidPrefixExceptionMapper caught an exception: {}", e.getMessage());
```

Comment:
You seem to have taken a different approach with this update than the main thrust of this PR. Was that intentional?

Reply:
Yeah, I removed it because it seemed like a client exception rather than a server exception. The approach I had was that client exceptions should be logged at debug and server exceptions should be warn (or higher).